UnityHttpClientsLibrary/BackOfficeHttpClient.cs

```diff
@@ -63,6 +63,13 @@ public string Debug_GetAuth()
     {
         return _headers["Authorization"];
     }
+
+    public IEnumerator GetGameConfig(Action<string> callback)
+    {
+        string path = "game-config";
+
+        return Post(path, _headers, new SessionPassword(gamePassword), callback);
```

Comment:
the session password is not required, it identifies the game through the player's authorisation token